spec/helpers/applications_helper_spec.rb

```diff
@@ -85,6 +85,12 @@
     end
   end
 
+  describe "decode" do
+    it "should decode the html entities such as &amp; to &" do
```

Comment:
@henare hmm I can't think of the case we don't want to decode it or potential security issues...any insights? I agree with looking into PlanningAlerts real world examples. I will look into it this week.